helm_deploy/values-dev.yaml

```diff
@@ -6,9 +6,6 @@ generic-service:
 
   ingress:
     host: visit-scheduler-dev.prison.service.justice.gov.uk
-    contextColour: green
```

Comment:
removed as not used on the new projects created from template projects and also mentioned here that it is not needed for the new helm charts - 
https://mojdt.slack.com/archives/C04JFG3QJE6/p1698751082281769?thread_ts=1698749911.096719&cid=C04JFG3QJE6